gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -317,11 +317,13 @@ public void extractMetadata(String schema, String entity, WorkUnit workUnit) thr
           JsonObject jsonObject = gson.fromJson(jsonStr, JsonObject.class).getAsJsonObject();
           targetSchema.add(jsonObject);
           headerColumns.add(targetColumnName);
+          sourceColumnName = "`" + sourceColumnName + "`";
```

Comment:
Different systems might have different escape characters. Please double check this, and probably we don't want to make a global change.